src/shared/components/plots/MultipleCategoryBarPlot.tsx

```diff
@@ -309,9 +309,14 @@ export default class MultipleCategoryBarPlot extends React.Component<IMultipleCa
             if (this.props.stacked) {
                 numBars = this.data[0].counts.length;
             } else {
-                numBars = _.sumBy(this.data, categoryPlotData => categoryPlotData.counts.length);
+                //majorCategories*minorCategories
+                numBars = this.data[0].counts.length * this.data.length;
+                //remove additional padding added by barSeparation
```

Comment:
wait if this is a problem then we should just remove barSeparation. I'm confused why would we add and then remove space?